### PR TITLE
[2.8] fix mocks

### PR DESCRIPTION
**Pull request by xabbuh**
| Q | A |
| --- | --- |
| Branch? | 2.8 |
| Bug fix? | yes |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets |  |
| License | MIT |
| Doc PR |  |

### CODE REVIEW

**Comment by xabbuh** [comment]
Tests are passing now. This one needs a rebase once the `2.7` branch is merged up.

File: phpunit [hunk 1]
```diff
@@ -11,7 +11,7 @@
  */
 
 // Please update when phpunit needs to be reinstalled with fresh deps:
-// Cache-Id-Version: 2016-03-23 14:50 UTC
+// Cache-Id-Version: 2016-03-25 09:45 UTC
```
Comment (xabbuh) [inline phpunit]: This is the same value as in the two fixed branches.

File: src/Symfony/Component/Form/Tests/Extension/Validator/Constraints/FormValidatorTest.php [hunk 2]
```diff
@@ -629,8 +629,11 @@ public function testNoViolationIfAllowExtraData()
 
         $context->expects($this->never())
             ->method('addViolation');
-        $context->expects($this->never())
-            ->method('addViolationAt');
+
+        if ($context instanceof Symfony\Component\Validator\ExecutionContextInterface) {
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Form/Tests/Extension/Validator/Constraints/FormValidatorTest.php]: Missing `\`?

Comment (xabbuh) [inline src/Symfony/Component/Form/Tests/Extension/Validator/Constraints/FormValidatorTest.php]: oops, good catch

**Comment by nicolas-grekas** [comment]
Thank you @xabbuh.

**Pull request closed without merging**